src/main/java/com/iexec/worker/compute/pre/PreComputeService.java

```diff
@@ -68,8 +68,6 @@ public PreComputeService(
      * If the task contains a dataset or some input files, the pre-compute enclave
      * is started to handle them.
      * 
-     * @param taskDescription
```

Comment:
Why do you remove @param instead of adding a description section like `@param myParam does blabla`?

Reply:
There are a lot of parameters to document and I usually don't exactly know what they are expected to be. Well, no more than what their names tell about them.
So, instead of writing information that wouldn't add value, I prefer to remove those parameters from the JavaDoc.